docs/api.rst

```diff
@@ -0,0 +1,36 @@
+Developer Interface for FoxPuppet
+*********************************
+
+This part of the documentation describes the interfaces for using FoxPuppet.
+
+.. py:module:: foxpuppet.foxpuppet
+
+.. autoclass:: FoxPuppet
+   :inherited-members:
+
+
+Window Manager
+==============================
+
+.. py:module:: foxpuppet.windows
+
+.. autoclass:: WindowManager
+  :inherited-members:
+
+
+Base Window Model
+=================
+
+.. py:module:: foxpuppet.windows.base
+
+.. autoclass:: BaseWindow
+  :inherited-members:
+
+
+Browser Model
+=============
+
+.. py:module:: foxpuppet.windows.browser
+
+.. autoclass:: BrowserWindow
+  :inherited-members:
```

Comment:
The structure should be a heading for windows and then a section for each window implementation. We shouldn't need to document the abstract base class.

```
# Windows
## Window Manager
## Browser Window
```